testing/trino-testing/src/main/java/io/trino/testing/AbstractTestQueryFramework.java

```diff
@@ -595,6 +595,17 @@ protected void assertNoDataRead(@Language("SQL") String sql)
                 results -> assertThat(results.getRowCount()).isEqualTo(0));
     }
 
+    protected void assertCatalogs(List<String> catalogs)
```

Comment:
Can we change the argument to `String...` so caller can omit `ImmutableList.of`?